app/components/Footer/index.css

```diff
@@ -1,52 +1,56 @@
 #Footer {
   background-color: var(--dark);
-  padding: 50px;
+  padding: 50px 0px 0px;
   text-align: center;
   width: 100%;
-  @media (max-width: 768px) {
-    padding: 50px 25px;
-  }
   & .footer-content {
     color: white;
-    display: flex;
-    flex-direction: row;
-    justify-content: space-between;
     margin: 0 auto;
     max-width: 1200px;
-    text-align: left;
     @media (max-width: 768px) {
       align-items: center;
-      flex-direction: column;
+    }
+    & img {
+      height: 22px;
+      width: 90px;
     }
     & .footer-links {
-      display: flex;
-      flex-direction: row;
-      max-width: 400px;
-      width: 100%;
       @media (max-width: 768px) {
         margin-bottom: 50px;
       }
-      & ul {
-        flex: 1;
-        margin: 0;
-        padding: 0;
+      & p {
+        color: lightgray;
+        font-size: 16px;
+        font-family: "Palanquin";
+        padding-bottom: 12px;
+        @media (max-width: 768px) {
+          padding-right: 30%;
+          padding-left: 30%;
+        }
+      }
```

Comment:
Wrong color (and in general, designers will never let you use a standard color like "lightgray" or "blue" 😆)

After clicking on the element in Figma, scrolling to the bottom of the "Inspect" panel will show the current styles as CSS values. We don't want to just copy/paste all of those, because Figma doesn't really know about "position" (everything is `absolute` to Figma), and we don't need the line-height because we only have 1 line (although we might need it for mobile?).

All that being said, the color we want here is `#A5ABB6`, and when we just write `lightgray` you get the [CSS "generic color" named "light gray"](https://www.computerhope.com/cgi-bin/htmlcolor.pl?c=D3D3D3), which is actually `#D3D3D3`.

In most cases, for our projects, we want to define all hexadecimal colors as variable names in `app/style.yml` so we can easily tweak styles later without a lot of find/replace. In our case here, `#A5ABB6` is already stored there as... "lightgray" 😅 

So either you learned a bit about colors and Figma here, or you already knew most of this and just messed up trying to reference that variable in `app/style.yml`. When we want to use those color varaibles, you have to wrap the name in the `var()` function, prefixed with two hyphens (`--`). In any case, the color value here should be:

```css
color: var(--lightgray);
```


<img width="1270" alt="Screen Shot 2022-10-12 at 4 54 33 PM" src="https://user-images.githubusercontent.com/1847581/195445880-5d651d14-95b7-4084-a694-d1ff77f59974.png">

Reply:
There also needs to be more margin between the logo `<img>` and this `<p>` (the design has them spaced a little further apart).